### DISCUSSION TITLE
Is it possible to verify based on publicKey that is a string?

### DISCUSSION THREAD

**Discussion by sandrooco** [panva/jose#180] [Q&A]
I need to verify a token from a 2nd party. They gave me a public key which is a string.

When I run it with a string I get the error `TypeError: invalid key input` because obviously a `string` is not a `Keylike`.
```
const jwt = 'ey...';
const publicKey = '...';

jwtVerify(jwt, publicKey, {
  issuer: 'x',
})
```

How can I "format/parse" it to work correctly?

**Comment by panva** [comment]
You create a [KeyLike](https://github.com/panva/jose/blob/v3.11.5/docs/types/types.keylike.md) from that ___-encoded key using whatever runtime's native functionality.
